appcell-resources/js/utils.js

```diff
@@ -36,7 +36,7 @@ ut.cellUrlWithEndingSlash = function(tempUrl, raiseError) {
  *     OR
  *     hello.png
  */
```

Comment:
1. Please comments for the new behaviour.  
1. notExtension -> withoutExtension